imu imu
bool[6] status
float32 temp
uint8 seq_num
float64 stamp